bittrex/bittrex.py

```diff
@@ -167,7 +165,7 @@ def get_markets(self):
         at Bittrex along with other meta data.
 
         1.1 Endpoint: /public/getmarkets
-        2.0 Endpoint: /pub/Markets/GetMarkets
+        2.0 NO Equivalent
```

Comment:
I am not so sure that these endpoints are permanently gone... but they are definitely broken now.  If they come back we can re-enable this functionality.